# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: js/src/tests/non262/extensions/typedarray.js
# Commit: 098569654f17
# Full Hash: 098569654f17fe958cfa7ca7077e4d770044a62e
# Author: Jon Coppeard <jcoppeard@mozilla.com>
# Date: 2022-10-28 21:22:11
# Regressor Bug: 1797755
# File Overlap Count: 1
# Description:
#   Bug 1797755 - Part 4: Remove option to set maximum mark stack capacity in release builds r=sfink
#   
#   Currently we have a GC parameter that allows setting a maximum mark stack
#   capacity. This is only ever used by test code, not in the browser. This
#   requires extra unnecessary work in release builds if we move to a different
# ==============================================================================

diff -r 6e5a935e576a -r 098569654f17 js/src/tests/non262/extensions/typedarray.js
--- a/js/src/tests/non262/extensions/typedarray.js	Fri Oct 28 09:47:14 2022 +0000
+++ b/js/src/tests/non262/extensions/typedarray.js	Fri Oct 28 09:47:14 2022 +0000
@@ -201,7 +201,9 @@
             }
         }
 
-        gcparam('markStackLimit', 200);
+        if (typeof setMarkStackLimit === 'function') {
+            setMarkStackLimit(200);
+        }
         var forceOverflow = [ buffer ];
         for (let i = 0; i < 1000; i++) {
             forceOverflow = [ forceOverflow ];
